docs/USER_GUIDE.md

```diff
@@ -89,16 +89,16 @@ Moveable`.
 
 ## Design patterns
 
-### `f :: X -> (SomeType #-> Unrestricted b) -> b` functions
+### `f :: X -> (SomeType #-> Ur b) -> b` functions
```

Comment:
Ah, there is a bit of a mistake here. The proper pattern is `f :: X -> (SomeType #-> Ur b) #-> Ur b` (fix it after you merge, it's unrelated to this PR).